### PR TITLE
Update lambda install instructions for lambda and cloudwatch

### PR DESCRIPTION
**Pull request by carkmorwin**

### CODE REVIEW

**Comment by carkmorwin** [comment]
Not in love with formatting, so any suggestions would be helpful.

**Comment by carkmorwin** [comment]
Merging this to be inline with current docs, will change things later.

File: docs/timber-platforms/aws-cloudwatch/installation.md [hunk 1]
```diff
@@ -10,38 +13,58 @@
...
      </tr>
      <tr>
-       <td style="text-align: right">S3 link URL</td>
-       <td>http://s3.amazonaws.com/packages.timber.io/cloudwatch-logs-lambda-function/timber-cloudwatch-logs-lambda-function-latest.zip</td>
+       <td style="text-align: right">Role Name</td>
+       <td><code>cloudwatchLogsToTimberRole<code></td>
+     </tr>
+     <tr>
+       <td style="text-align: right">Policy Templates</td>
+       <td><code>Basic Edge Lambda Permissions<code></td>
```
Comment (jsonmaur) [inline docs/timber-platforms/aws-cloudwatch/installation.md]: `</code>`

File: docs/timber-platforms/aws-cloudwatch/installation.md [hunk 2]
```diff
@@ -10,38 +13,58 @@
...
-       <td style="text-align: right">Code entry type</td>
-       <td><code>Upload a file from Amazon S3</code></td>
+       <td style="text-align: right">Role</td>
+       <td><code>Create new role from template(s)</code></td>
      </tr>
      <tr>
-       <td style="text-align: right">S3 link URL</td>
-       <td>http://s3.amazonaws.com/packages.timber.io/cloudwatch-logs-lambda-function/timber-cloudwatch-logs-lambda-function-latest.zip</td>
+       <td style="text-align: right">Role Name</td>
+       <td><code>cloudwatchLogsToTimberRole<code></td>
```
Comment (jsonmaur) [inline docs/timber-platforms/aws-cloudwatch/installation.md]: `</code>`

Comment (carkmorwin) [reply docs/timber-platforms/aws-cloudwatch/installation.md]: thanks, I'll double check things now

**Comment by jsonmaur** [comment]
I like the tables. Is there a reason to use the html over [markdown tables](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#tables)? Definitely think we should have a custom component to support nice tables like this without having to write out all the HTML... It would go a long way.

File: docs/timber-platforms/aws-cloudwatch/installation.md [hunk 3]
```diff
@@ -10,38 +13,58 @@
...
-       <td>Assign an IAM user with basic lambda execution permissions.</td>
+       <td style="text-align: right">S3 link URL</td>
+       <td><a href=http://s3.amazonaws.com/packages.timber.io/cloudwatch-logs-lambda-function/timber-cloudwatch-logs-lambda-function-latest.zip>http://s3.amazonaws.com/packages.timber.io/cloudwatch-logs-lambda-function/timber-cloudwatch-logs-lambda-function-latest.zip</a></td>
      </tr>
-   </table>
+  </table>
+
+3\. Add TIMBER_API_KEY as an environment variable with value {{my-timber-api-key}}
+
+OR
```
Comment (jsonmaur) [inline docs/timber-platforms/aws-cloudwatch/installation.md]: Might help to give this a bit more seperation, such as `###### _OR_`

**Comment by carkmorwin** [comment]
@jsonmaur That's just what we had before, so I wanted to follow the same styling. That being said we should probably just use markdown.

**Comment by carkmorwin** [comment]
These changes are in master

**Comment by carkmorwin** [comment]
These changes are in master

**Pull request merged**